### ISSUE TITLE
Failing with Fava 1.23

### ISSUE THREAD

**Issue by mmoya** [polarmutex/fava-envelope#58]
Deprecated `FavaLegder.entries` was removed in Fava 1.23 and fava-envelope uses it:

```
...
  File "/home/mmoya/.cache/pypoetry/virtualenvs/mmoya-ledger-YHEMojRP-py3.10/lib/python3.10/site-packages/fava_envelope/__init__.py", line 26, in get_budgets_months_available
    self.generate_budget_df(currency)
  File "/home/mmoya/.cache/pypoetry/virtualenvs/mmoya-ledger-YHEMojRP-py3.10/lib/python3.10/site-packages/fava_envelope/__init__.py", line 19, in generate_budget_df
    self.ledger.entries,
AttributeError: 'FavaLedger' object has no attribute 'entries'
```

downgrading Fava back to 1.22.3 fix the issue.

**Comment by polarmutex** [comment]
working on a fix, as soon as I can, been really busy lately

**Comment by polarmutex** [comment]
released 0.5.5 which should fix the issue(tested locally), reopen if something else breaks
